### PR TITLE
Divyanshu | Test | Support of US tax in Giddh - Bugs

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvhk6yx

### CODE REVIEW

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts [hunk 1]
```diff
@@ -310,6 +314,12 @@ export class PurchaseRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true,
             },
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
+            if (activeCompany) {
+                this.activeCompanyCountryCode = activeCompany.countryV2?.alpha2CountryCode;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts:320]: confirm with @ravindersinghdang sir for not using component store for this or not

File: apps/web-giddh/src/assets/css/global/tax-authority.scss [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
-            &.bordered-table {
-                tr {
-                    th,td {
-                        border: 1px solid var(--color-table-primary-content-bar-border) !important;
-                        
-                        &:not(:first-child) {
-                            text-align: right;
-                        }
+            tr {
+
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/assets/css/global/tax-authority.scss]: remove blank space

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/global/tax-authority.scss (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/assets/css/global/tax-authority.scss]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts [hunk 3]
```diff
@@ -310,6 +314,12 @@ export class PurchaseRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true,
             },
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts:318]: use component store

File: apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.ts [hunk 4]
```diff
@@ -291,6 +295,12 @@ export class SalesRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true
             }
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.ts:299]: use component store

File: apps/web-giddh/src/app/shared/create-address/create-address.component.ts [hunk 5]
```diff
@@ -83,6 +91,11 @@ export class CreateAddressComponent implements OnInit, OnDestroy {
      */
     public ngOnInit(): void {
         this.setFormData();
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/create-address/create-address.component.ts:94]: use component store

File: apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.html [hunk 6]
```diff
@@ -7,9 +7,11 @@
         <dropdown-field [options]="months" (selectedOption)="getMonthStartAndEndDate($event)"
             [placeholder]="commonLocaleData?.app_select_month" [defaultValue]="selectedMonth?.label"></dropdown-field>
     </div>
-    <div class="input-group custom-datepicker custom-new-datepicker" (click)="showGiddhDatepicker($event)">
-        <input type="text" name="selectedDateRange" [value]="selectedDateRangeUi ? selectedDateRangeUi : ''"
-            class="giddh-datepicker date-range-picker border-radius-2" />
+    <div>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.html:10]: what's the use of this div?

File: apps/web-giddh/src/assets/locale/mr.json [hunk 7]
```diff
@@ -528,6 +528,8 @@
     "app_bulk_export": "मोठ्या प्रमाणात निर्यात",
     "app_pin_code": "पिन कोड",
     "app_enter_pin_code": "पिन कोड प्रविष्ट करा",
+    "app_zip_code": "पिनकोड",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/mr.json]: use correct translation

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/locale/mr.json (hunk 7 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/assets/locale/mr.json]: This correct I have checked it again on google translation and chatGPT too

File: apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.html (hunk 6 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.html:10]: Inside this date picker div will take all available space which causes datepicker show correct width but, its wrapper will take full space so if click on that div then also datepicker get opened, so as solution to this I wrapper that in side div

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts:318]: Sir, here no component store is implemented that's why I use global store.

File: apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.ts (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.ts:299]: Sir, here no component store is implemented that's why I use global store.

File: apps/web-giddh/src/app/shared/create-address/create-address.component.ts (hunk 5 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/shared/create-address/create-address.component.ts:94]: Sir, here no component store is implemented that's why I use global store.

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts:320]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**